packages/core/src/docdb/commands/addRegion.ts

```diff
@@ -0,0 +1,167 @@
+            }
+
+            if (node.cluster.DBClusterMembers?.length === 0) {
+                void vscode.window.showErrorMessage(
+                    localize(
+                        'AWS.docdb.addRegion.noInstances',
+                        'Cluster must have at least one instance to add a region'
+                    )
+                )
+                throw new ToolkitError('Cluster must have at least one instance to add a region', { cancelled: true })
+            }
+
+            const unsupportedInstanceFound = node.instances.find(
+                (instance) => !isSupportedGlobalInstanceClass(instance.DBInstanceClass!)
+            )
+
+            if (unsupportedInstanceFound) {
+                void vscode.window.showErrorMessage(
+                    localize(
+                        'AWS.docdb.addRegion.unsupportedInstanceClass',
+                        'Instance class {0} not supported for global cluster.  Upgrade the instances then try again.',
+                        unsupportedInstanceFound.DBInstanceClass
+                    )
+                )
+                throw new ToolkitError('Instance class not supported for global cluster', {
+                    cancelled: true,
+                    code: 'docdbInstanceClassNotSupported',
+                })
+            }
+        } else {
+            globalClusterName = node.cluster.GlobalClusterIdentifier
+
+            if (node.cluster.GlobalClusterMembers!.length > 4) {
+                void vscode.window.showErrorMessage(
+                    localize('AWS.docdb.addRegion.maxRegions', 'Global clusters can have a maximum of 5 regions')
+                )
+                throw new ToolkitError('Global clusters can have a maximum of 5 regions', {
+                    cancelled: true,
+                    code: 'docdbMaxRegionsInUse',
+                })
+            }
+        }
+
+        if (node.cluster.Status !== 'available') {
+            void vscode.window.showErrorMessage(
+                localize('AWS.docdb.deleteCluster.clusterStopped', 'Cluster must be running')
+            )
+            throw new ToolkitError('Cluster not available', { cancelled: true, code: 'docdbClusterStopped' })
+        }
+
+        const wizard = new CreateGlobalClusterWizard(node.regionCode, node.cluster.EngineVersion, node.client, {
+            initState: { GlobalClusterName: globalClusterName },
+        })
+        const response = await wizard.run()
+
+        if (!isValidResponse(response)) {
+            throw new CancellationError('user')
+        }
+
+        const regionCode = response.RegionCode
+        let input: CreateDBClusterMessage
+        let clusterName = response.GlobalClusterName
+
+        try {
+            if (node instanceof DBClusterNode) {
+                // Create new global cluster from regional cluster
+                const primaryCluster = node.cluster
+
+                getLogger().info(`docdb:Creating global cluster: ${clusterName}`)
```

Comment:
I'd add a space between all these:
```suggestion
                getLogger().info(`docdb: Creating global cluster: ${clusterName}`)
```